cid-redirects.json

```diff
@@ -3872,6 +3872,9 @@
   "/docs/cse/automation-service/automation-service-bridge": "/docs/platform-services/automation-service/automation-service-bridge",
   "/docs/cse/automation-service/automation-service-integration-framework": "/docs/platform-services/automation-service/automation-service-integration-framework",
   "/docs/send-data/collect-from-other-data-sources/kubernetes": "/docs/send-data/kubernetes",
+  "/docs/send-data/collect-from-other-data-sources/azure-blob-storage": "/docs/send-data/collect-from-other-data-sources/azure-blob-storage/block-blob/collect-logs",
```

Comment:
@JV0812 This is the line causing the error. It's redirecting from a file that already exists to another file that already exists. So you'd need to remove or correct this line.